cypress.config.js

```diff
@@ -25,4 +25,5 @@ module.exports = defineConfig({
 		// do not retry in `cypress open`
 		openMode: 0,
 	},
+	"numTestsKeptInMemory": 1,
```

Comment:
Oooof, we should all reduce that number, yes! :see_no_evil:

Reply:
I had issues with heap overflows when testing this. Problem also is heap does not get cleaned up when rerunning a test or moving to the next.

This also means loosing the ability to inspect previously run tests - but the benefit still outweights the cost for me here.

I'll still increase this to 5. One test typically takes up 40 MB of heap space. So 5 are 200 MB - so heap is full after 10 test runs.